ansible/files/postgresql_config/supautils.conf.j2

```diff
@@ -7,7 +7,7 @@ supautils.drop_trigger_grants = '{"postgres":["auth.audit_log_entries","auth.ide
 # omitted because doesn't require superuser: pgmq
 # omitted because protected:                 plpgsql
 supautils.privileged_extensions = 'address_standardizer, address_standardizer_data_us, autoinc, bloom, btree_gin, btree_gist, citext, cube, dblink, dict_int, dict_xsyn, earthdistance, fuzzystrmatch, hstore, http, hypopg, index_advisor, insert_username, intarray, isn, ltree, moddatetime, orioledb, pg_buffercache, pg_cron, pg_graphql, pg_hashids, pg_jsonschema, pg_net, pg_prewarm, pg_repack, pg_stat_monitor, pg_stat_statements, pg_tle, pg_trgm, pg_walinspect, pgaudit, pgcrypto, pgjwt, pgroonga, pgroonga_database, pgrouting, pgrowlocks, pgsodium, pgstattuple, pgtap, plcoffee, pljava, plls, plpgsql_check, plv8, postgis, postgis_raster, postgis_sfcgal, postgis_tiger_geocoder, postgis_topology, postgres_fdw, refint, rum, seg, sslinfo, supabase_vault, supautils, tablefunc, tcn, timescaledb, tsm_system_rows, tsm_system_time, unaccent, uuid-ossp, vector, wrappers'
-supautils.privileged_extensions_custom_scripts_path = '/etc/postgresql-custom/extension-custom-scripts'
+supautils.extension_custom_scripts_path = '/etc/postgresql-custom/extension-custom-scripts'
```

Comment:
Updating the config because `extension_custom_scripts_path` (previously unset aka `null`) were [overwriting](https://github.com/supabase/supautils/blob/ba6566a4a112cc179d6d2d77ca04769cde07147e/src/supautils.c#L1271-L1289) `privileged_extensions_custom_scripts_path` (previously set to the path).

Postgres doesn't like multiple custom variables using the same underlying [static variable](https://github.com/supabase/supautils/blob/ba6566a4a112cc179d6d2d77ca04769cde07147e/src/supautils.c#L43). We should just remove the deprecated configs in the future.